src/app/starredentries/starredentries.component.html

```diff
@@ -16,110 +16,254 @@
               </a>
-            </h3>
-            <span mat-line>
-              <small class="spacing">Last updated {{ tool?.lastUpdated | date }}</small>
-              <span class="spacing" *ngFor="let type of tool?.descriptorType">{{ type | uppercase }}</span>
-              <a class="spacing" *ngIf="tool?.provider" [href]="tool?.providerUrl">{{ tool?.provider }}</a>
-              <a class="spacing" *ngIf="tool?.imgProvider" [href]="tool?.imgProviderUrl">{{ tool?.imgProvider }}</a>
-            </span>
-            <app-starring class="pull-right" style="width: 100px" [tool]="tool" (starGazersChange)="starGazersChange()"></app-starring>
-            <mat-divider></mat-divider>
-          </mat-list-item>
-        </mat-list>
+              <app-starring class="pull-right" [workflow]="workflow" (starGazersChange)="starGazersChange()"></app-starring>
+            </div>
+            <mat-card-header>
+              <div *ngIf="workflow?.starredUsers.length !== 0" fxLayout="row" fxLayoutAlign="end center"></div>
+              <mat-card-subtitle class="overflow">{{ workflow?.description }}</mat-card-subtitle>
+            </mat-card-header>
+            <div>
+              <hr />
+              <div fxLayout="row" fxLayoutAlign="space-between stretch">
+                <div fxLayout="row" fxLayoutAlign="flex-start" fxLayoutGap="1rem">
+                  <small class="spacing">Last updated {{ workflow?.lastUpdated | date }}</small>
+                  <div fxLayout="row wrap" fxLayoutGap="1rem">
+                    <span>
+                      <mat-chip-list>
+                        <mat-chip class="workflow-label-tag" *ngFor="let label of workflow?.labels | slice: 0:3">{{
+                          label.value
+                        }}</mat-chip>
+                        <mat-chip class="gray-label-tag" *ngIf="workflow.verified === true">
+                          <span class="verified-check" fxLayoutGap="0.5rem" fxLayoutAlign=" center">
+                            <mat-icon class="verified-icon">done</mat-icon>
+                            <span>Verified</span>
+                          </span>
+                        </mat-chip>
+                        <mat-chip class="gray-label-tag">
+                          {{ workflow?.descriptorType | uppercase }}
+                        </mat-chip>
+                        <mat-chip class="gray-label-tag" *ngIf="workflow.verified === true">
+                          <span class="verified-check" fxLayoutGap="0.5rem" fxLayoutAlign=" center">
+                            <mat-icon class="verified-icon">done</mat-icon>
+                            <span>Verified</span>
+                          </span>
+                        </mat-chip>
+                      </mat-chip-list>
+                    </span>
+                  </div>
+                </div>
+                <a routerLink="/workflows/{{ workflow.full_workflow_path }}" target="_blank" rel="noopener" class="no-underline">
```

Comment:
https://github.com/dockstore/dockstore-ui2/pull/635#discussion_r282653584

Reply:
That comment is for an external link, outside dockstore.org. This comment is for a link within dockstore.

Reply:
Ok, so your process for determining whether a link should be opened in a new tab depends on whether it's external or not? My initial assumption based on that PR is that you wanted every link in the starred page to open in a new tab (since no specific reason was given)

Reply:
Yes, I would go with that. :) And I think we generally behave like that already; just tried a couple of links on the home page for a not vary rigorous confirmation.

Reply:
So first of all, that's not part of the frontend style guide yet (I was planning to add it if further thought and research corroborates it). It's not really consistent depending on what's considered internal or not (gui assets like pdfs, dockstore documentation open in new tab).  Looking into it, I think the summary of https://www.nngroup.com/articles/new-browser-windows-and-tabs/ was what I've always thought for links which is more based on the task the user is trying to accomplish and internal vs external is not really a good way of determining it.

So for this specific case (because it used to not open into a new tab, then it was changed to a new tab)... before I change it back to no new tab, I need some kind of good detailed comment to prevent some new team member from stumbling upon it and changing it back to a new tab